debian/control

```diff
@@ -59,7 +59,8 @@ Build-Depends: debhelper-compat (= 12), dh-python, python3:any, dracut-core, qui
                libgcc-s1,
                squashfs-tools,
                snapd (>= 2.50+20.04),
-               golang-go, indent, libapparmor-dev, libcap-dev, libfuse-dev, libglib2.0-dev, liblzma-dev, liblzo2-dev, libseccomp-dev, libudev-dev, openssh-client, pkg-config, python3, python3-docutils, python3-markdown, squashfs-tools, tzdata, udev, xfslibs-dev
+               systemd-bootchart,
```

Comment:
the `golang-go...` line is a carbon copy of the build-deps that snapd declares, because we are temporarily building snap-bootstrap fork, because snapd has not yet reviewed/merged changes that core-initrd relies on.

Thus it would be nice to keep that whole line unchanged for now, even though it has duplicate build-depends (squashfs-tools) declared.